inc/functions/cdn.php

```diff
@@ -55,11 +55,11 @@ function get_rocket_cdn_url( $url, $zone = array( 'all' ) ) {
 		if ( strpos( $parse_url['path'], $home ) === false && ! preg_match( '#(' . $wp_content_dirname . '|wp-includes)#', $parse_url['path'] ) ) {
 			return $url;
 		} else {
-			$path = str_replace( $home, '', ltrim( $parse_url['path'], '//' ) );
+			$parse_url['path'] = str_replace( $home, '', ltrim( $parse_url['path'], '//' ) );
```

Comment:
Shouldn't be `'/'`?

Reply:
I modified this to fix the notice, but reading the code I don't even know what it is supposed to do (that's old code initially)